core/src/main/scala/org/apache/spark/scheduler/CoarseCookSchedulerBackend.scala

```diff
@@ -329,32 +337,28 @@ class CoarseCookSchedulerBackend(
       }
 
       def handleDisconnectedExecutor(executorId: String): Unit = {
-        logInfo(s"Recieved disconnect message from executor with ID: ${executorId}")
+        logInfo(s"Received disconnect message from executor with ID: $executorId")
```

Comment:
It would be great if we could log the instanceId of the disconnected executor somewhere (since the executorId is no longer the same as instanceId) so the user can quickly query cook for the failed executor. If this is too much work though feel free to punt on it. We probably should audit the logging patterns here at some point.